src/build/strings.ts

```diff
@@ -28,7 +28,7 @@ const strings = {
   'extensionHost.snippet.launch.label': 'VS Code Extension Development',
 
   'edge.useWebView.description':
-    "(Edge (Chromium) only) When 'true', the debugger will treat the runtime executable as a host application that contains a WebView allowing you to debug the WebView script content. When set to 'advanced', the debugger will wait to connect to a WebView matching the urlFilter.",
+    "(Edge (Chromium) only) When 'true' or 'advanced', the debugger will treat the runtime executable as a host application that contains a WebView allowing you to debug the WebView script content.",
```

Comment:
If they're now the same, do we need two options or can it just be true/false?

Reply:
Good point, I was keeping `advanced` around for backwards compatibility with vcode-edge-debug2 configs but it will be less confusing for users if we just drop `advanced`.

I will update this PR to do that!

Reply:
Sounds good, thanks. For this extension we don't care about back compat with edge-debug (only for node and chrome debug)

Reply:
Done!